lib/trino-hive-formats/src/main/java/io/trino/hive/formats/HiveFormatUtils.java

```diff
@@ -264,18 +263,15 @@ public static List<String> getTimestampFormatsSchemaProperty(Map<String, String>
             char c = property.charAt(position);
             if (c == TIMESTAMP_FORMATS_ESCAPE) {
                 // the next character must be an escape or separator
-                checkArgument(
-                        position + 1 < property.length(),
-                        "Invalid '%s' property value '%s': unterminated escape at end of value",
-                        TIMESTAMP_FORMATS_KEY,
-                        property);
+                if (position + 1 >= property.length()) {
+                    // TODO: include in the error message the origin of this property (eg, a Hive table from whose storage we read the property) so the user can find and fix it.
```

Comment:
I don't think that we actually want to do this.  The caller can wrapper the exception and provide context.